### PR TITLE
Add a clock dashboard with a digital clock display for different timezones

### PR DESCRIPTION
**Pull request by nainar**
This patch adds a clock dashboard with a digital clock display for different timezones.

The times are formatted as:
If it is a weekend -> the day is shown in red
If it is working hours in the timezone -> the time is shown green
If it is not working hours in the timezone -> the time is shown red.

Also modifies the blink.json to add the timezones the team cares about.

PTAL?

Thanks!

### CODE REVIEW

File: client/cz-clock-dash.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        seconds = (seconds < 10) ? "0" + seconds : seconds;
+
+        if (day != 0 && day != 6)
+          return this.formattedTime(hours, minutes, seconds, meridian);
+        else
+          return "<span style='color: red'>" + this.intToDay(day) + "</span>";
+      },
+      mutateTimeDay: function(timezone, idx) {
+        this.set('timezones.' + idx + '.data', this.currentTimeDay(timezone));
+        this.async(()=>this.mutateTimeDay(timezone, idx), 500);
```
Comment (dstoc) [inline client/cz-clock-dash.html]: why is this async?

Comment (dstoc) [inline client/cz-clock-dash.html]: I think you should format this as `...async(() => this...`

File: client/cz-clock-dash.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        width: 300px;
+      }
+    </style>
+
+    <paper-card id='card' heading='Time around the world'>
+      <div class="card-content">
+        <div class='card-flex'>
+          <template is="dom-repeat" items="{{timezones}}">
+            <paper-item-body two-line>
+              <div id="timer" inner-h-t-m-l=[[item.data]]></div>
```
Comment (dstoc) [inline client/cz-clock-dash.html]: Why set innerhtml? You should be able to set up the `<span>` using data binding and use a `class` attribute for styling: `<span class="{{item.data.class}}">{{item.data.time}}</span>`

File: client/cz-clock-dash.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          type: 'Object',
+          notify: true,
+          value: function() { return []; },
+          observer: "timezonesChanged"
+        }
+      },
+      attached: function() {
+        registerSource('cz-config', 'timezones', function(timezones) {
+          this.set('timezones', timezones);
+        }.bind(this));
```
Comment (dstoc) [inline client/cz-clock-dash.html]: using an arrow function should avoid the need to bind here

File: client/cz-clock-dash.html [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+            </paper-item-body>
+          </template>
+        </div>
+      </div>
+    </paper-card>
+  </template>
+
+  <script>
+    Polymer({
+      is: "cz-clock-dash",
```
Comment (dstoc) [inline client/cz-clock-dash.html]: google javascript style [prefers single quotes over double quotes](https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings)

File: client/cz-clock-dash.html (hunk 1 above)
Comment (dstoc) [inline client/cz-clock-dash.html]: > why is this async?

Oh, this is the loop. 500ms seems a bit fast? Don't we only need to update once a minute? It also appears that when the timezones in the source change we will end up with multiple loops running.

File: client/cz-clock-dash.html (hunk 2 above)
Comment (nainar) [inline client/cz-clock-dash.html]: Done.

File: client/cz-clock-dash.html (hunk 4 above)
Comment (nainar) [inline client/cz-clock-dash.html]: Done.

File: client/cz-clock-dash.html (hunk 3 above)
Comment (nainar) [inline client/cz-clock-dash.html]: Done.

File: client/cz-clock-dash.html (hunk 1 above)
Comment (nainar) [inline client/cz-clock-dash.html]: Done.

**Comment by nainar** [comment]
@dstockwell, Made the changes you asked for, PTAL?

File: configs/platform.json [hunk 5]
```diff
@@ -13,5 +13,12 @@
   {"user": "Trent", "email": "tapted@chromium.org"},
   {"user": "Raymes", "email": "raymes@chromium.org"}
 ],
+"timezones": [
+  {"city": "MTV", "utc": "-7"},
+  {"city": "NYC", "utc": "-4"},
+  {"city": "LON", "utc": "0"},
```
Comment (dstoc) [inline configs/platform.json]: London and Munich are the same number of characters as Sydney.

File: client/cz-clock-dash.html [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+        }
+        else {
+          data = {timeString: this.intToDay(day), class: 'notWorkingHours'};
+        }
+        console.log(data.timeString, data.class);
+        return data;
+      },
+      mutateTimeString: function() {
+        this.timezones.forEach((timezone, idx) => this.set('timezones.' + idx + '.data', this.currentTimeDay(timezone)));
+        this.async(()=>this.mutateTimeString(), 1000);
```
Comment (dstoc) [inline client/cz-clock-dash.html]: spaces

**Comment by dstoc** [comment]
lgtm

**Comment by nainar** [comment]
Changes made since last lgtm:
1. Changed timezone formatting in platform.json
2. Switched to 24 hour clock
3. Removed seconds
4. Added space -> formatting error.

**Comment by dstoc** [comment]
For future reference, here's an API to go from lat+long to time offset: https://maps.googleapis.com/maps/api/timezone/json?location=-33.8650,151.2094&timestamp=1331161200

**Pull request merged**